.htaccess

```diff
@@ -53,7 +53,7 @@ RewriteRule ^__version__$ version.json [L]
 RewriteRule ^__heartbeat__$ heartbeat.cgi [L]
 
 # New single page interface for filing bugs
-RewriteRule ^new[-_]bug$ new_bug.cgi [L,QSA]
+RewriteRule ^new[-_]bug($|/) new_bug.cgi [L,QSA]
```

Comment:
Yes, to accept other extended URLs. Plan is to have something like `new-bug/Core` file a bug in Core.